modules/core/src/v2/wallets.ts

```diff
@@ -336,10 +341,26 @@ export class Wallets {
       }
 
       if (!_.isUndefined(params.gasPrice)) {
-        if (!_.isNumber(params.gasPrice)) {
+        const gasPriceBN = new BigNumber(params.gasPrice);
```

Comment:
can we also throw if gasPrice is not a number or a string? 
want to avoid weird BigNum output if we pass gasPrice as a boolean or something

Reply:
I modified the message, and it appeared correctly in the cases of boolean and string with letters.